### PR TITLE
Mutualize fetching functions

### PR DESCRIPTION
**Pull request by Meklo**
Need #2187

Depends on gridsuite/commons-ui#455

### CODE REVIEW

File: src/services/utils.js [hunk 1]
```diff
@@ -5,139 +5,13 @@
...
-                    text
-            );
-            error.status = response.status;
-        }
-        throw error;
-    });
-};
+import { fetchAppsAndUrls, fetchEnv } from '@gridsuite/commons-ui';
 
 export const getToken = () => {
```
Comment (flomillot) [inline src/services/utils.js:10]: It's still needed ?

**Review by flomillot — COMMENTED** [review]
Need to update commons-ui or it won't works no ?

File: src/services/utils.js (hunk 1 above)
Comment (flomillot) [reply src/services/utils.js:10]: Well it's the point of the ticket, mutualize what we can, it won't be done after if we don't do it now, i'm pretty sure, so well...
You can check with a TL but for me, it need to be done here alos.

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]
Found that
* `src/services/explore.ts`
  - `createFilter()` not removed
  - `elementExists()` not removed
* `commons-ui` version not updated in `package.json`

File: src/services/utils.js (hunk 1 above)
Comment (Meklo) [reply src/services/utils.js:17]: I've just talked about it with @TheMaskedTurtle, we ought to prioritize fixing commons-ui to release gridstudy deadlock.
I can create a following ticket to clean up  the rest of utils functions getVersion, auth function, etc...

Comment (Meklo) [reply src/services/utils.js:17]: This said I can clean up `getToken` since it's already in commons ui, didn't noticed earlier

Comment (Meklo) [reply src/services/utils.js]: Yes I figured this too later, fine by me I'll do a second PR later then

**Review by dbraquart — COMMENTED** [review]
code: ok
tests: to be done after merge wConclicts

**Pull request closed without merging**